scripts/random_vector.py

```diff
@@ -29,7 +29,7 @@ def error(msg):
 
 (minx, miny, maxx, maxy) = map(float, options.extent.split(","))
 
-driverName = "ESRI Shapefile"
+driverName = "GPKG"
```

Comment:
Revert this change too - it's a purely test based tool, switching to gpkg has no benefit here

Reply:
changed back